From cd933c16c968f6209c198ca14ceedce18c6e3ca3 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Sat, 20 Oct 2018 06:16:05 +0200
Subject: [PATCH 175/293] xlat: add TCP_CLOSE to netlink_states

* xlat/netlink_states.in: Decode state 7 as TCP_CLOSE, since that's what
net core sets socket's initial state to.
---
 xlat/netlink_states.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xlat/netlink_states.in b/xlat/netlink_states.in
index 4d4aee7..d2ffb7b 100644
--- a/xlat/netlink_states.in
+++ b/xlat/netlink_states.in
@@ -1,2 +1,4 @@
 NETLINK_UNCONNECTED	0
 NETLINK_CONNECTED	1
+/* The initial socket state, as set by net/code/sock.c:sock_init_data */
+TCP_CLOSE		7
-- 
1.7.12.4

